core/controllers/base_test.py

```diff
@@ -269,6 +271,31 @@ def test_i18n_keys(self):
                     self.log_line('- %s' % key)
                 self.log_line('')
 
+    def alphabetic_test_i18n_keys(self):
```

Comment:
The function name has to begin with `test`, for example `test_alphabetic_i18n_keys`, otherwise the test will not be run.